components/bootloaders/STM/stm32f103/include/CAN.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
```

Comment:
(nit) Needs file documentation and headers

Reply:
yeah all these files are going to need that